schema.sql

```diff
@@ -8,3 +8,6 @@ CREATE TABLE animals (
 	neutered bool,
 	weight_kg decimal
 );
+
+ALTER TABLE animals
+ADD COLUMN species varchar(20);
```

Comment:
- [ OPTIONAL] Good one setting limit to 20. However, in cases where the name of the species is longer than the set value, saving the data will be a problem. Therefore, you can try to sometimes go with a max value like 255.

Reply:
- [OPTIONAL] You can also consider updating the author section of the readme to match your details.